benchmark_utils/gd_inner.py

```diff
@@ -2,42 +2,6 @@
 from functools import partial
 
 
-def gd_inner(inner_oracle, inner_var, outer_var, step_size,
-             n_steps=1):
-    """
-    Implement gradient descent on the inner oracle.
-
-    Parameters
-    ----------
-    inner_oracle : oracle class
-        Inner oracle.
-
-    inner_var : array
-        Initial value of the inner variable.
-
-    outer_var : array
-        Value of the outer variable.
-
-    step_size : float
-        Step size of the gradient descent.
-
-    n_steps : int
-        Number of steps of the gradient descent.
-
-    Returns
-    -------
-    inner_var : array
-        Value of the inner variable after n_steps of gradient descent.
-    """
-    for i in range(n_steps):
-        grad_inner = inner_oracle.grad_inner_var(
-            inner_var, outer_var, slice(None)
-        )
-        inner_var -= step_size * grad_inner
-
-    return inner_var
-
-
 @partial(jax.jit, static_argnames=('grad_inner', 'n_steps'))
```

Comment:
should we remove `jit`?

Reply:
Since is used with non stochastic algo (and so with non jitted steps), it is worth to jit it